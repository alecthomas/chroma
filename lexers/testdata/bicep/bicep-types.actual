type fizz = string

type buzz = fizz[]

type pop = {
  krispies: 'snap'|'crackle'|'pop'
}