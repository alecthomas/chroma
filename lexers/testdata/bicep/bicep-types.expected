[
  {"type":"KeywordDeclaration","value":"type"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameVariable","value":"fizz"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"="},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameVariable","value":"string"},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"KeywordDeclaration","value":"type"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameVariable","value":"buzz"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"="},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameVariable","value":"fizz"},
  {"type":"Punctuation","value":"[]"},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"KeywordDeclaration","value":"type"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameVariable","value":"pop"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"="},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"NameVariable","value":"krispies"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"'snap'"},
  {"type":"Punctuation","value":"|"},
  {"type":"LiteralString","value":"'crackle'"},
  {"type":"Punctuation","value":"|"},
  {"type":"LiteralString","value":"'pop'"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"Punctuation","value":"}"}
]
